### PR TITLE
Bug 1134040 - Do not update direction on subject input field any more.

### PR DESCRIPTION
**Pull request by autra**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[autra Augustin Trancart (autra)](https://github.com/autra) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=114346175f84)

File: apps/sms/js/subject_composer.js [hunk 1]
```diff
@@ -62,17 +62,6 @@
     }
   }
 
-  /** Without bug 1103011 fixed, subject's caret would not be set at correct
-   * position with dir attribute set to auto. So we need to specify the auto
-   * direction only when subject is not empty.
-   */
-  function updateDirection(value) {
```
Comment (azasypkin) [inline apps/sms/js/subject_composer.js]: Please remove unit test that depends on this logc as well [1]. It's failing currently on your PR.

[1] https://github.com/mozilla-b2g/gaia/blob/c79b08251a6d43953f067d9acc185504d4d27142/apps/sms/test/unit/subject_composer_test.js#L212

**Comment by try-server-hook** [comment]
[autra Augustin Trancart (autra)](https://github.com/autra) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=192b54ce4459)

**Comment by try-server-hook** [comment]
[autra Augustin Trancart (autra)](https://github.com/autra) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=86a583c6626a)

**Pull request merged**